lib/platform/bitbucket/index.ts

```diff
@@ -247,8 +247,7 @@ export async function deleteBranch(
 async function isPrConflicted(prNo: number): Promise<boolean> {
   const diff = (
     await bitbucketHttp.get(
-      `/2.0/repositories/${config.repository}/pullrequests/${prNo}/diff`,
-      { json: false } as any
```

Comment:
This is wrong for our http layer, maybe forgotten while refactored because of anoying `any` cast. 🙃